conformance/failure_list_cpp.txt

```diff
@@ -17,6 +17,8 @@ Recommended.Proto3.JsonInput.FieldNameDuplicate
 Recommended.Proto3.JsonInput.FieldNameDuplicateDifferentCasing1
 Recommended.Proto3.JsonInput.FieldNameDuplicateDifferentCasing2
 Recommended.Proto3.JsonInput.FieldNameNotQuoted
+Recommended.Proto3.JsonInput.IgnoreUnknownEnumStringValueInMapValue.ProtobufOutput
+Recommended.Proto3.JsonInput.IgnoreUnknownEnumStringValueInRepeatedField.ProtobufOutput
```

Comment:
Why were these added as failures?  What's the expected behavior here?  From the PR description it sounds like C++ and Java were implemented "correctly", so it's a little strange that two of the added tests are failing for C++

Reply:
I think this was committed already and is causing presubmits to fail since these indeed shouldn't be in the failure list: e.g. https://github.com/protocolbuffers/upb/actions/runs/3473202608/jobs/5804957297

Reply:
It was, I'm trying to retroactively understand the goal here

Reply:
(sorry, i was actually wrong above -- the presubmit failure was for upb and wasn't from the cpp failure list)

Reply:
@zhangskz, based on your latest message I suspect you already executed the following steps to debug `upb` failure, but to over-communicate adding it here.

I can reproduce the failure you linked above [link](https://github.com/protocolbuffers/upb/actions/runs/3472397745/jobs/5803170896) locally. The error message is:

```
These tests were listed in the failure list, but they don't exist:

  Recommended.Proto3.JsonInput.IgnoreUnknownEnumStringValueInMapValue.ProtobufOutput
  Recommended.Proto3.JsonInput.IgnoreUnknownEnumStringValueInRepeatedField.ProtobufOutput
```

It looks like the conformance test runner and the failure list fell out of sync:
* conformance test runner was was taken from d938afd6e35676f6b8c8012d9eacd1dc0bb693b4 (defined in [`bazel/workspace_deps.bzl`](https://github.com/protocolbuffers/upb/blob/main/bazel/workspace_deps.bzl#L27))
* failure list is taken from 064b263c42274dbf849a6df026c6e7f2bc7e2828 (copied to [`upb/conformance_upb_failures.txt`](https://github.com/protocolbuffers/upb/blob/main/upb/conformance_upb_failures.txt))

There is an open PR that is supposed to fix this: https://github.com/protocolbuffers/upb/pull/938. That PR is held back due to a [python test failure](https://github.com/protocolbuffers/upb/actions/runs/3476676730/jobs/5812112814) that looks unrelated to the conformance test that we added.